azure-pipelines-official.yml

```diff
@@ -169,6 +170,28 @@ extends:
             artifactName: 'NGen Logs'
             publishLocation: Container
 
+          # Publish an artifact that the RoslynInsertionTool is able to find by its name.
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact VSSetup'
+            condition: succeeded()
+            targetPath: 'artifacts\VSSetup\$(BuildConfiguration)'
+            artifactName: 'VSSetup'
+
+          # Publish our NuPkgs as an artifact. The name of this artifact must be PackageArtifacts as the
+          # arcade templates depend on the name.
+          - output: pipelineArtifact
+            displayName: 'Publish Artifact Packages'
+            condition: succeeded()
+            targetPath: 'artifacts\packages\$(BuildConfiguration)'
```

Comment:
Do we want to standardize on either `'artifacts\...'` or `'$(Build.SourceDirectory\artifacts\...'`?